doc/changes/latest.inc

```diff
@@ -224,3 +228,5 @@ API
 - The ``threshold`` argument in :meth:`mne.preprocessing.ICA.find_bads_ecg` defaults to ``None`` in version 0.21 but will change to ``'auto'`` in 0.22 by `Yu-Han Luo`_
 
 - The default argument ``meg=True`` in :func:`mne.pick_types` will change to ``meg=False`` in version 0.22 by `Clemens Brunner`_
+
+- Add ``file_format`` parameter to :func:`mne.read_surface` and :func:`mne.write_surface` by `Marijn van Vliet`_
```

Comment:
This shouldn't actually go here as it's a new feature, not something that someone will need to change to keep their code working as it did before (unless they are using very strange filenames), so probably better just to combine this with your Wavefront entry above